### PR TITLE
A few more mappings

### PR DESCRIPTION
**Pull request by Blayyke**

### CODE REVIEW

File: mappings/net/minecraft/client/particle/ParticleManager.mapping [hunk 1]
```diff
@@ -2,3 +2,4 @@ CLASS dn net/minecraft/client/particle/ParticleManager
 	FIELD a level Lfd;
 	FIELD c textureManager Lji;
 	FIELD d rand Ljava/util/Random;
+	METHOD a addBlockBreakParticles (IIIII)V
```
Comment (valoeghese) [inline mappings/net/minecraft/client/particle/ParticleManager.mapping]: addTileBreakParticles

File: mappings/net/minecraft/item/ItemType.mapping [hunk 2]
```diff
@@ -117,11 +117,19 @@ CLASS gm net/minecraft/item/ItemType
 	METHOD a setContainerItem (Lgm;)Lgm;
 		ARG 1 itemType
 	METHOD a getTranslationKey (Liz;)Ljava/lang/String;
-	METHOD a activate (Liz;Lfd;Lgs;)Liz;
-	METHOD a interact (Liz;Lgs;Lfd;IIII)Z
+	METHOD a postMine (Liz;IIIILls;)Z
+	METHOD a use (Liz;Lfd;Lgs;)Liz;
+	METHOD a inventoryTick (Liz;Lfd;Lsn;IZ)V
+	METHOD a useOnBlock (Liz;Lgs;Lfd;IIII)Z
```
Comment (valoeghese) [inline mappings/net/minecraft/item/ItemType.mapping]: useOnTile

File: mappings/net/minecraft/client/particle/ParticleManager.mapping (hunk 1 above)
Comment (Blayyke) [reply mappings/net/minecraft/client/particle/ParticleManager.mapping]: Fixed

File: mappings/net/minecraft/item/ItemType.mapping (hunk 2 above)
Comment (Blayyke) [reply mappings/net/minecraft/item/ItemType.mapping]: Fixed

**Pull request merged**